app/widgets/Chat/Chat.php

```diff
@@ -1014,7 +1012,7 @@ private function prepareComposeList(array $list)
      */
     private function validateJid($jid)
     {
-        return (Validator::stringType()->noWhitespace()->length(6, 60)->validate($jid));
+        return prepJid($jid);
```

Comment:
You can simply remove that method and directly call prepJid ;) Same with the others.